Index: src/org/ulpgc/is1/model/Customer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.ulpgc.is1.model;\r\n\r\nimport java.util.ArrayList;\r\n\r\npublic class Customer {\r\n    private String name;\r\n\r\n    private ArrayList<Address> address;\r\n\r\n    public String getName() {\r\n        return name;\r\n    }\r\n\r\n    public void setName(String name) {\r\n        this.name = name;\r\n    }\r\n\r\n    public String getSurname() {\r\n        return surname;\r\n    }\r\n\r\n    public void setSurname(String surname) {\r\n        this.surname = surname;\r\n    }\r\n\r\n    private String surname;\r\n\r\n    public void addAddress(String street, int number, int postalCode, String city) {\r\n        Address a = new Address(street, number, postalCode, city);\r\n        address.add(a);\r\n    }\r\n\r\n    public Customer(String name, String surname){\r\n\r\n        this.name = name;\r\n        this.surname = surname;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/org/ulpgc/is1/model/Customer.java b/src/org/ulpgc/is1/model/Customer.java
--- a/src/org/ulpgc/is1/model/Customer.java	(revision 560d61e518238138525fc69150ef7d263d8d1b4b)
+++ b/src/org/ulpgc/is1/model/Customer.java	(date 1697367202422)
@@ -29,10 +29,14 @@
         Address a = new Address(street, number, postalCode, city);
         address.add(a);
     }
+    //creamos una nueva lista
+    private ArrayList<Order> orders;
 
-    public Customer(String name, String surname){
+
+    public Customer(){
 
         this.name = name;
         this.surname = surname;
+        this.orders = new ArrayList<Order>();
     }
 }
Index: src/org/ulpgc/is1/model/Order.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.ulpgc.is1.model;\r\n\r\nimport java.util.ArrayList;\r\n\r\npublic class Order {\r\n\r\n    private final int id;\r\n\r\n    private static int NEXT_ID = 0;\r\n\r\n    private Customer customer;\r\n\r\n    private ArrayList<Customer>c;\r\n\r\n    private ArrayList<OrderItem>orderItem;\r\n\r\n    public Order(Customer customer){\r\n        this.id = NEXT_ID++;\r\n        this.customer = new Customer();\r\n        this.c = new ArrayList<Customer>();\r\n    }\r\n\r\n    public void addOrderItem(int quantity, Dish dish){\r\n        OrderItem orderItem1 = new OrderItem(quantity, dish);\r\n        orderItem.add(orderItem1);\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/org/ulpgc/is1/model/Order.java b/src/org/ulpgc/is1/model/Order.java
--- a/src/org/ulpgc/is1/model/Order.java	(revision 560d61e518238138525fc69150ef7d263d8d1b4b)
+++ b/src/org/ulpgc/is1/model/Order.java	(date 1697367201834)
@@ -3,25 +3,15 @@
 import java.util.ArrayList;
 
 public class Order {
-
-    private final int id;
-
     private static int NEXT_ID = 0;
-
-    private Customer customer;
-
-    private ArrayList<Customer>c;
+    private final int id;
 
-    private ArrayList<OrderItem>orderItem;
-
-    public Order(Customer customer){
-        this.id = NEXT_ID++;
-        this.customer = new Customer();
-        this.c = new ArrayList<Customer>();
-    }
-
-    public void addOrderItem(int quantity, Dish dish){
+    private ArrayList<OrderItem> orderItem;
+    public void addOrderItem(int quantity, Dish dish) {
         OrderItem orderItem1 = new OrderItem(quantity, dish);
         orderItem.add(orderItem1);
     }
+    public Order() {
+        this.id = NEXT_ID++;
+    }
 }
